src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -48,30 +51,24 @@ public ValidationFailure Test(NotifiarrSettings settings)
                 SendNotification(variables, settings);
                 return null;
             }
-            catch (HttpException ex)
+            catch (NotifiarrException ex)
             {
-                if (ex.Response.StatusCode == HttpStatusCode.Unauthorized)
-                {
-                    _logger.Error(ex, "API key is invalid: " + ex.Message);
-                    return new ValidationFailure("APIKey", "API key is invalid");
-                }
-
-                _logger.Error(ex, "Unable to send test message: " + ex.Message);
-                return new ValidationFailure("APIKey", "Unable to send test notification");
+                return new ValidationFailure("APIKey", ex.Message);
             }
             catch (Exception ex)
             {
-                _logger.Error(ex, "Unable to send test notification: " + ex.Message);
-                return new ValidationFailure("", "Unable to send test notification");
+                _logger.Error(ex, ex.Message);
+                return new ValidationFailure("", "Unable to send test notification. Check the log for more details.");
             }
         }
 
         private void ProcessNotification(StringDictionary message, NotifiarrSettings settings)
         {
             try
             {
-                var requestBuilder = new HttpRequestBuilder(URL).Post();
-                requestBuilder.AddFormParameter("api", settings.APIKey).Build();
+                var instanceName = _configFileProvider.InstanceName;
+                var requestBuilder = new HttpRequestBuilder(URL + "/api/v1/notification/prowlarr/" + settings.APIKey).Post();
```

Comment:
@austinwbest wont agree, but these should use `x-api-key` headers to keep the api key out of logs/urls.

Reply:
it's cleansed anyway; would only be cleartext logged to console.

Reply:
Any upstream proxy can capture the url.

Reply:
TBD future problem